src/test/groovy/TestUtils/XMLUtil.groovy

```diff
@@ -0,0 +1,16 @@
+package TestUtils
+
+import static org.junit.Assert.assertEquals
+
```

Comment:
@hofi1 Could you please add javadoc for this class?

Reply:
@zhurlik sure